views/help.html

```diff
@@ -35,6 +36,10 @@ <h1 class="title"><%= __('Help') %></h1>
   'systemInfoDataError': __('Sorry, there was an error, please try again later.'),
   'systemStatusCronStatusTitle': __('Cron'),
   'systemStatusQueueTitle': __('Sending Queue'),
+  'yourPrivacyContent1': __("MailPoet respects your privacy. We don't track any information about your website or yourself without your explicit consent."),
```

Comment:
Could you use typographical apostrophes in all your strings? I mean those `’` instead of `'`

Reply:
Done